### PR TITLE
Avoid using mockito in akka-cluster-sharding

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: akka-cluster-metrics/src/test/scala/akka/cluster/metrics/AbstractSigarProxy.scala [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package akka.cluster.metrics class AbstractSigarProxy {
```
Comment (patriknw) [inline akka-cluster-metrics/src/test/scala/akka/cluster/metrics/AbstractSigarProxy.scala]: is this used?

**Review by patriknw — COMMENTED** [review]
+1

File: akka-cluster-metrics/src/test/scala/akka/cluster/metrics/AbstractSigarProxy.scala (hunk 1 above)
Comment (raboof) [reply akka-cluster-metrics/src/test/scala/akka/cluster/metrics/AbstractSigarProxy.scala]: oops, leftover from an attempt to also remove it from cluster-metrics tests

**Review by patriknw — APPROVED** [review]
LGTM

File: akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingInternalsSpec.scala [hunk 2]
```diff
@@ -32,35 +30,49 @@ class ClusterShardingInternalsSpec extends AkkaSpec("""
     |akka.actor.provider = cluster
     |akka.remote.classic.netty.tcp.port = 0
     |akka.remote.artery.canonical.port = 0
-    |""".stripMargin) with MockitoSugar {
+    |""".stripMargin) {
```
Comment (helena) [inline akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingInternalsSpec.scala]: YES. I was surprised to see that used every time I was in this file.

File: project/Dependencies.scala [hunk 3]
```diff
@@ -208,8 +208,7 @@ object Dependencies {
         Provided.levelDBNative,
         Test.junit,
         Test.scalatest.value,
-        Test.commonsIo,
-        Test.mockito)
```
Comment (helena) [inline project/Dependencies.scala]: 👍

**Review by helena — APPROVED** [review]
Thanks

**Pull request merged**